ui/app/mainui/activitycenter/views/ActivityNotificationBase.qml

```diff
@@ -32,8 +32,7 @@ Item {
         anchors.right: parent.right
         anchors.left: parent.left
         messageTimestamp: notification.timestamp
-        previousMessageTimestamp: root.activityCenterStore.activityCenterList.getNotificationData(
-                                    previousNotificationIndex, "timestamp")
+        previousMessageTimestamp: filteredIndex === 0 || !notification.previousTimestamp ? 0 : notification.previousTimestamp
```

Comment:
And this is needed because we always want to render timestamp label for the first item even when we have filter enabled